src/renderer/components/cluster-manager/cluster-view.tsx

```diff
@@ -87,7 +92,7 @@ export class ClusterView extends React.Component {
     const { clusterId, cluster, isReady } = this;
 
     if (cluster && !isReady) {
-      return <ClusterStatus clusterId={clusterId} className="box center"/>;
+      return <ClusterStatus key={clusterId} clusterId={clusterId} className="box center"/>;
```

Comment:
Unique key so that `ClusterStatus` is not cached between clusters.

Reply:
Are we only using a single iframe now?

Reply:
This is not related to iframes at all. We are still having iframe per cluster. But we have just one status/error page that was shared between clusters -> in some cases it showed logs etc from previous cluster. `key` makes React to recreate the component instance on change.

Reply:
Oh okay makes sense. I thought `ClusterStatus` was within the iframe.